docs/Release.md

```diff
@@ -16,6 +16,19 @@ Follow this list for items that must be completed for release of the `vscode-has
   - Github actions will automatically release the extension to VSCode- and VSX-Marketplace.
   - If you want to create a pre-release, create a [pre-release in Github](https://github.com/haskell/vscode-haskell/releases). The github action will perform the appropriate actions automatically and publish the pre-release of the extension to VSCode- and VSX-Marketplace.
 
+## Branching policy
+
+Sometimes there is a release (stable) and pre-release (unstable) at the same time and we need to do a release for the stable release and sometimes we need to do a release for the pre-release series.
+To simplify the release management, the following policy is in place:
+
+* The branch `master` contains the current stable release
```

Comment:
Perhaps I don't understand how this works here, but isn't the usual approach that master is always the "development" branch (which looks more like a pre-release) and you make an explicit release branch for stable releases? Is this due to some magic where we tell vscode to follow a branch for new releases or something?

Reply:
There is no magic involved. So far, we didn't choose a particular branching/release strategy and master was equal to develop, and at the same time the release branch. Development is slow, but currently we have two branches that require maintenance, hence a policy change. Are you proposing that master becomes the pre-release branch, and we add a branch, say `release/2.4`, where we add commits which are backported from master?

The only thing to keep in mind, https://marketplace.visualstudio.com/items?itemName=haskell.haskell shows the README from the master branch, afaict.

Reply:
Yeah, that's what I'm suggesting. It's what I've seen in most other software projects, but :shrug: 

Normally what you do is that you decide you're cutting a release, and then you make a release branch to which you backport stuff if you need to, and then you eventually tag off the release branch.

Reply:
The policy has been updated, thanks for your input!